test/composition/default-declarative-shadow-dom.html

```diff
@@ -185,6 +207,25 @@ <h2>Custom Shadow DOM alternative composition</h2>
         });
 
     });
+    describe('starcounter-include when imports a template that contains <code>template is="declarative-shadow-dom"</code> with <code>presentation</code> attribute other than `author`, when imports a template', function () {
+        const REFERENCE_COMPOSITION = REFERENCE_HOST;
```

Comment:
Please rename `REFERENCE_HOST` to `REFERENCE_PARENT` throughout the project so that we use consistent naming.